CHANGELOG.md

```diff
@@ -30,7 +30,9 @@
 - Spawn more threads for CPU intensive work. ([#1378](https://github.com/getsentry/relay/pull/1378))
 - Add missing fields to DeviceContext ([#1383](https://github.com/getsentry/relay/pull/1383))
 - Improve performance of Redis accesses by not running `PING` everytime a connection is reused. ([#1394](https://github.com/getsentry/relay/pull/1394))
-- Change to the internals of the store actor. ([#1397](https://github.com/getsentry/relay/pull/1397))
+- Distinguish between various discard reasons for profiles. ([#1395](https://github.com/getsentry/relay/pull/1395))
+- Add missing fields to GPUContext ([#1391](https://github.com/getsentry/relay/pull/1391))
```

Comment:
this accidentally moved to the wrong section?

Reply:
i mean, the line below this :)

Reply:
I think it should be in the Internal section? (At least that is where the entry for the change of the Healthcheck actor is as well)